src/drivers/api.ts

```diff
@@ -154,22 +154,27 @@ export const fetchWithStreaming = async function* <T>(
         break;
       }
       const value = read.value;
-      const text = textDecoder.decode(value);
-      const parsedResponse = JSON.parse(text);
-      if (parsedResponse.type === "error") {
-        throw new Error(parsedResponse.error);
+      const jsonlSegments = convertJsonlToArray<{
```

Comment:
I'm not sure this fully solves the problem. I think it relies on the assumption that when reading from the reader(), the individual chunks that the reader gives you are going to be auto-chunked into jsonl segments, but I think the chunking can be arbitrary.

I was running into this chunking issue when working on other CLI stuff (I was on a Windows VM, but that may not be relevant). I can try to see if I can repro that.